client/apps/edit/components/admin/test/index.coffee

```diff
@@ -22,6 +22,7 @@ describe 'EditAdmin', ->
         EditAdmin::setupAuthorAutocomplete = sinon.stub()
         EditAdmin::setupFairAutocomplete = sinon.stub()
         EditAdmin::setupPartnerAutocomplete = sinon.stub()
+        EditAdmin::setupAuctionAutocomplete = sinon.stub()
```

Comment:
I didn't see tests for the other ones, but I'm happy to write them if you see fit!